config/application.rb

```diff
@@ -15,8 +15,9 @@ class Application < Rails::Application
     #
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
-    #
-    # config.time_zone = "Central Time (US & Canada)"
+
+    config.time_zone = "Kolkata"
```

Comment:
required?

Reply:
Sir I want to store time in db in IST as I want to make comparisons to the current time often in the app.